roles/openstack-setup/tasks/computes.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: create host aggs
+  os_nova_host_agg:
+    name: "{{ hostvars[item]['compute_ag'] }}"
+    az: "{{ hostvars[item]['compute_ag'] }}"
+    state: present 
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
+  run_once: True
+  when: hostvars[item]['compute_ag'] is defined
+  with_items: "{{ groups['compute'] }}"
+
+- name: assign hosts to aggs
```

Comment:
How are we going to deal with host already in different ag? either user error on initial run or customer driven change.